shell/client/admin/preinstalled-apps.html

```diff
@@ -0,0 +1,108 @@
+<template name="newAdminPreinstalledApps">
+  <h1>
+    <ul class="admin-breadcrumbs">
+      <li>{{#linkTo route="newAdminRoot"}}Admin{{/linkTo}}</li>
+      <li>Pre-installed Apps</li>
+    </ul>
+  </h1>
+
+  {{#if hasSuccess}}
+    {{#focusingSuccessBox}}
+      {{message}}
+    {{/focusingSuccessBox}}
+  {{/if}}
+  {{#if hasError}}
+    {{#focusingErrorBox}}
+      {{message}}
+    {{/focusingErrorBox}}
+  {{/if}}
+  <p>
+    Apps you configure on this page to be pre-installed will automatically be installed (show under /apps) for new users.
+  </p>
+  <form class="admin-preinstalled-apps">
+    <div class="form-group">
+      <label>
+        Productivity Suite
+      </label>
+      <span class="form-subtext">
+        We recommend pre-installing these core apps that are useful to most users.
+      </span>
+      <div class="productivity-suite" role="grid">
```

Comment:
suggest renaming `productivity-suite` to `suite` if you're going to reuse it for system apps